tests/ert/unit_tests/shared/test_port_handler.py

```diff
@@ -257,112 +186,25 @@ def test_def_passive_live_nok_nok_close_ok_ok(unused_tcp_port):
     # we want to try again, so close it
     sock.close()
 
-    sock = find_available_socket(
-        custom_range=custom_range,
-        custom_host="127.0.0.1",
-        will_close_then_reopen_socket=True,
-    )
+    sock = find_available_socket(port_range=port_range, host="127.0.0.1")
     _, port = sock.getsockname()
     assert port == unused_tcp_port
     assert sock is not None
     assert sock.fileno() != -1
 
 
-def test_reuse_active_close_nok_ok(unused_tcp_port):
+def test_socket_can_not_rebind_if_open(unused_tcp_port):
     """
     Executive summary of this test
 
-    1. the original socket is obtained with will_close_then_reopen_socket=True
-    2. activity is triggered on the socket using a dummy-server/client
-    3. socket is closed
-    4. port can not be re-bound in default mode (TIME_WAIT?)...
-    5. ... but can with will_close_then_reopen_socket=True (ignoring TIME_WAIT)
-    """
-    custom_range = range(unused_tcp_port, unused_tcp_port + 1)
-
-    # Note: Setting will_close_then_reopen_socket=True on original socket
-    orig_sock = find_available_socket(
-        custom_range=custom_range,
-        custom_host="127.0.0.1",
-        will_close_then_reopen_socket=True,
-    )
-    host, port = orig_sock.getsockname()
-    assert port == unused_tcp_port
-    assert orig_sock is not None
-    assert orig_sock.fileno() != -1
-
-    # Run a dummy-server to actually use the socket a little, then close it
-    _simulate_server(host, port, orig_sock)
-    orig_sock.close()
-
-    # Using will_close_then_reopen_socket=False fails...
-    with pytest.raises(NoPortsInRangeException):
-        find_available_socket(custom_range=custom_range, custom_host="127.0.0.1")
-
-    # ... but using will_close_then_reopen_socket=True succeeds
-    sock = find_available_socket(
-        custom_range=custom_range,
-        custom_host="127.0.0.1",
-        will_close_then_reopen_socket=True,
-    )
-    host, port = sock.getsockname()
-    assert port == unused_tcp_port
-    assert sock is not None
-    assert sock.fileno() != -1
-
-
-def test_reuse_active_live_nok_nok(unused_tcp_port):
-    """
-    Executive summary of this test
-
-    1. the original socket is obtained with will_close_then_reopen_socket=True
+    1. the original socket is obtained
```

Comment:
prefixing with `Executive summary of this test` is redundant, it should just get to the point.

Reply:
made a new pr for that. #10007